### ISSUE TITLE
Support for Python 3.8 runtime for az functionapp create

### ISSUE THREAD

**Issue by dmpe** [Azure/azure-cli#13772]
**Is your feature request related to a problem? Please describe.**

In Portal you can create azure function with python 3.8. support. However, this is missing in azure cli, causing 

```
az functionapp create -n xxx -c eastus -g xxxx -s xxxx --disable-app-insights true --os-type linux --runtime python --runtime-version 3.8 --tags reason=function

--runtime-version 3.8 is not supported for the selected --runtime python. Supported versions are: 3.6, 3.7
```

**Describe the solution you'd like**

Support in azure cli for python 3.8. version

**Describe alternatives you've considered**

using portal

**Comment by yonzhan** [comment]
functionapp

**Comment by anthonychu** [comment]
@dmpe Can you please try adding `--functions-version 3` to your command and try again. By default, `az functionapp create` currently creates a Functions V2 app and V2 only supports Python 3.6 and 3.7. Thanks.

**Comment by dmpe** [comment]
Gotha, thank you. I am looking forward to having Functions V3 by default :)

**Comment by anthonychu** [comment]
Because folks might be using the CLI for automation, we don't want their scripts to break by changing because we changed the default. So there's a good chance that the version will have to stay explicitly set. In hindsight, it probably should have been a required parameter.

**Comment by dmpe** [comment]
OK, understand. Maybe an "important" section in the documentation could be created/highlighted somewhere. and func cli help page (`-h`) could reference it :)

**Comment by kamperiadis** [comment]
Marking this issue as resolved since Python 3.8 is already supported. You can obtain the supported runtimes by running `az functionapp list-runtimes`
